src/macros.rs

```diff
@@ -1,10 +1,305 @@
+//! Extra macro documentation.
+
 #[doc(hidden)]
 pub use fizyr_rpc_macros::interface as interface_impl;
 
 #[macro_export]
 /// Define an RPC interface.
+///
+/// This macro generates a module with a `Client` and `Server` struct,
+/// and some more helper types.
+///
+/// The client struct can be used to initiate requests and send stream messages.
+/// It can be created from a [`PeerWriteHandle`] or a [`PeerHandle`].
+/// Note that if you create the client from a [`PeerHandle`], the [`PeerReadHandle`] will not be accessible.
```

Comment:
Might be interesting to say why, unless it is too complicated of course.

Reply:
Well, it's just a consequence of how the API works. There's no specific reason other than there is no logical way to expose the read handle when consuming the `PeerHandle`.